### DISCUSSION TITLE
[SOLVED] Comma-separated digit groups in the file size

### DISCUSSION THREAD

**Discussion by arsinclair** [doublecmd/doublecmd#1449] [Suggestions]
Hey everyone! Not sure if this is already implemented, but after 20 minutes searching in the settings I wasn't able to find this option.

It would be great to be able to use a comma/dot as a file size separator when the size is displayed in bytes. Similarly to the Total Commander. (Please note, I'm not talking about a decimal separator, but a digit group separator)

Currently Double Commander will display the size like this:

![image](https://github.com/doublecmd/doublecmd/assets/2878904/1fee19d8-c315-4146-b2c7-5b78a885c5cc)

And Total Commander uses a comma:

![image](https://github.com/doublecmd/doublecmd/assets/2878904/e876d876-b03e-4778-bbba-fbb6c68a176c)

If this is something that can already be configured, I'd appreciate if you point me in the right direction.

Peace!

**Comment by alexx2000** [comment]
Hi, DC always use "space" as separator.

**Reply by arsinclair** [reply]
Okay, thank you for confirming. So would it be possible to make it configurable?

**Reply by arsinclair** [reply]
Or maybe at least it can respect the OS-wide regional settings?

![image](https://github.com/doublecmd/doublecmd/assets/2878904/7a53bb68-11ff-4c65-bf98-693309836d4f)

**Comment by alexx2000** [comment]
> Or maybe at least it can respect the OS-wide regional settings?

Done (revision 11203).

**Reply by arsinclair** [reply]
🤩 that was incredibly fast! Thanks a lot! And thanks for such a great tool.

**Reply by arsinclair** [reply]
I didn't find any credentials, but do you accept donations?

**Reply by alexx2000** [reply]
No.

**Reply by arsinclair** [reply]
Thank you again.
